Quick/Reporting/XCTestObservationCenter+Reporting.m

```diff
@@ -0,0 +1,159 @@
+//
+//  XCTestObservationCenter+Reporting.m
+//  Quick
+//
+//  Created by Paul Young on 1/2/15.
+//  Copyright (c) 2015 Brian Ivan Gesiak. All rights reserved.
+//
+
+#import "XCTestObservationCenter+Reporting.h"
+#import <XCTest/XCTestRun.h>
+#import "XCTestDriver.h"
+#import <objc/runtime.h>
+
+@implementation XCTestObservationCenter (Reporting)
+
++ (void)load {
+    NSDictionary *environment = [[NSProcessInfo processInfo] environment];
+    NSString *reporter = [environment objectForKey:@"QUICK_REPORTER"];
```

Comment:
This is awesome. At some point, let's have this access an attribute on `QuickConfiguration`, and have configuration attributes override-able via the command line.

I've never been able to get command line arguments working, so I'm afraid I don't have any advice on how to get things working for the iOS test target.